.idea/modules.xml

```diff
@@ -2,8 +2,9 @@
 <project version="4">
   <component name="ProjectModuleManager">
     <modules>
-      <module fileurl="file://$PROJECT_DIR$/HealthOutLoud.iml" filepath="$PROJECT_DIR$/HealthOutLoud.iml" />
+      <module fileurl="file://C:\Users\court\AndroidStudioProjects\health-out-loud\HealthOutLoud.iml" filepath="C:\Users\court\AndroidStudioProjects\health-out-loud\HealthOutLoud.iml" />
```

Comment:
Android Studio automatically added these lines, and I'm not sure how to make it so they're generic. For now everyone might have to change the filepaths when they update these changes